packages/flutter_test/lib/src/binding.dart

```diff
@@ -373,11 +373,33 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   /// [AutomatedTestWidgetsFlutterBinding] implementation to increase the
   /// current timeout, if any. See [AutomatedTestWidgetsFlutterBinding.addTime]
   /// for details.
+  ///
+  /// If your test code depends on the result of a task that did not complete,
+  /// the most common symptom would be a hanging test as FakeAsync environment
+  /// is unable to resolve a future that was created in `runAsync`. If you
+  /// observe such behavior or flakiness, you have a number of options:
+  ///
+  /// - Consider restructuring your code so you do not need `runAsync`. This is
```

Comment:
nit: I believe we use `*` instead of `-` throughout the code base for enumerations.